cmd/entrypoint/main.go

```diff
@@ -53,6 +54,13 @@ func main() {
 		PostWriter:      &realPostWriter{},
 		Results:         strings.Split(*results, ","),
 	}
+
+	// Copy any creds injected by creds-init into the $HOME directory of the current
+	// user so that they're discoverable by git / ssh.
+	if err := credentials.CopyCredsToHome(credentials.CredsInitCredentials); err != nil {
```

Comment:
any reason why it is required here ? (in the `entrypoint` wrapper binary)

Reply:
`entrypoint` is the only tool we have that for-sure runs in the Step container, with access to (and knowledge of) the Step's HOME dir.

So we can only really use entrypoint to perform this copy I think.  And the nice benefit is that the behaviour now matches precisely before and after the feature flag: before the flag the credentials end up in $HOME (creds-init writes them there) and after the flag the credentials end up in $HOME (entrypoint writes them there).

Reply:
ah very good point :+1: gotcha :wink: